cmd/gaia/app/app.go

```diff
@@ -133,6 +133,7 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 		app.keySlashing,
 		&stakingKeeper, app.paramsKeeper.Subspace(slashing.DefaultParamspace),
 		slashing.DefaultCodespace,
+		app.bankKeeper,
```

Comment:
Just to clarify, we are adding the ability to access bank keeper to the slashing keeper.

Reply:
Technically yes and no. The slashing keeper requires an interface contract `GetSendEnabled` -- this is the only thing the slashing module can call or use.